src/worker/core/draft/genPlayersWithoutSaving.ts

```diff
@@ -33,12 +33,11 @@ const genPlayersWithoutSaving = async (
 		forceScrubs = numRealPlayers > 0.5 * normalNumPlayers;
 	}
 
-	const baseAge =
-		bySport({
-			hockey: 18,
-			default: 19,
-		}) -
-		(draftYear - g.get("season"));
+	const draftAge = g.get("draftAge");
+	const baseAge = !isSport("football")
+		? draftAge[0] - (draftYear - g.get("season"))
+		: draftAge[0] - 2 - (draftYear - g.get("season"));
```

Comment:
I think it would be slightly clearer to do:

```
// The distribution of football rookies in the default FBGM was written to assume they were generated at 19 and then developed for 2 seasons. If `draftAge` existed when that code was written, it would not have made sense to include those 2 seasons. But doing something about that is much more difficult, so we want to keep the 2 seasons of extra development for FBGM currently.
let baseAge = draftAge[0] - (draftYear - g.get("season"));
if (isSport("football")) {
    baseAge -= 2;
}
```

Then you can never make some typo that pulls your two `baseAge` formulas out of sync.

Reply:
But I really should say... good work handling this tricky situation, and at first glance it looks like you handled it correctly. My comments are just about what will make this easiest to understand when somebody comes back to this code next year :)